tools/osx/crosstool/BUILD.toolchains

```diff
@@ -1,5 +1,6 @@
 package(default_visibility = ["//visibility:public"])
 
+load("@local_config_platform//:constraints.bzl", "HOST_CONSTRAINTS")
```

Comment:
This works but I assume this is safe to add since the C++ logic does this too https://github.com/bazelbuild/bazel/blob/8a42645ec500874b0440475763ab680d5efc1e6a/tools/cpp/BUILD.toolchains.tpl#L1